### PR TITLE
T1622

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/4fd72c6d5c2aa97/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/4fd72c6d5c2aa97/output.txt

Total script time: 0.74 mins

Server is running on http://pr564.butyr.org/templates/test.html

File: templates/supported/newscaster/butter/saved-data.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
       "id": "Media0",
       "name": "Media0",
       "url": [
-        "http://videos-origin.mozilla.org/serv/webmademovies/Moz_Doc_0329_GetInvolved.mp4",
-        "http://videos-origin.mozilla.org/serv/webmademovies/Moz_Doc_0329_GetInvolved.webm"
+        "http://videos-cdn.mozilla.org/serv/webmademovies/Moz_Doc_0329_GetInvolved.mp4",
```
Comment (secretrobotron) [inline templates/supported/newscaster/butter/saved-data.json]: These are wrong :(.

**Pull request merged**